pom.xml

```diff
@@ -463,6 +463,31 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>io.quarkus</groupId>
+        <artifactId>quarkus-maven-plugin</artifactId>
+        <version>1.6.1.Final</version>
+        <executions>
+          <execution>
+            <goals>
+              <goal>build</goal>
+            </goals>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <groupId>org.jboss.jandex</groupId>
+        <artifactId>jandex-maven-plugin</artifactId>
+        <version>1.0.7</version>
+        <executions>
+          <execution>
+            <id>make-index</id>
+            <goals>
+              <goal>jandex</goal>
+            </goals>
+          </execution>
+        </executions>
+      </plugin>
```

Comment:
I meant to move it to the `<pluginManagement>` section in order to define the versions and config.
We then still need to include the plugins in the <build>` sectionof the modules that should run them but can then omit the version and config ...

Reply:
👍 Fixed now